src/Plivo/Resources/CallFeedback/CallFeedbackInterface.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Plivo\Resources\CallFeedback;
+
+use Plivo\Exceptions\PlivoValidationException;
+use Plivo\Exceptions\PlivoResponseException;
+use Plivo\BaseClient;
+use Plivo\Resources\ResourceInterface;
+use Plivo\Resources\ResourceList;
+
+use Plivo\Resources\ResponseUpdate;
+use Plivo\Util\ArrayOperations;
+
+/**
+ * Class CallFeedbackInterface
+ * @package Plivo\Resources\CallFeedback
+ */
+class CallFeedbackInterface extends ResourceInterface
+{
+    const CALLINSIGHTS_BASE_URL = "https://stats.plivo.com";
+    const CALLINSIGHTS_API_VERSION = "v1";
+    /**
+     * CallFeedbackInterface constructor.
+     * @param BaseClient $plivoClient
+     * @param string $authId
+     */
+    function __construct(BaseClient $plivoClient)
+    {
+        parent::__construct($plivoClient);
+        $this->uri = "v1/Call/";
+    }
+
+    public function create($callUUID, $rating, $issues=[], $notes="")
+    {
+        $mandatoryArgs = [
+            'callUUID' => $callUUID,
+            'rating' => $rating
+        ];
+
+        $optionalArgs = [];
+        if (count($issues) > 0) {
+            $optionalArgs['issues'] = $issues;
+        }
+
+        if (strlen($notes) > 0) {
+            $optionalArgs['notes'] = $notes;
+        }
+
+        $optionalArgs['isCallInsightsRequest'] = TRUE;
+        $requestPath = sprintf("Call/%s/Feedback/", $callUUID);
+        $optionalArgs['CallInsightsEndpoint'] = sprintf("%s/%s/%s", self::CALLINSIGHTS_BASE_URL, self::CALLINSIGHTS_API_VERSION, $requestPath);
+
+        if (ArrayOperations::checkNull($mandatoryArgs)) {
+            throw new PlivoValidationException(
```

Comment:
if any of one mandatory param is not given or empty, then also we should throw an exception.
like if 
```
$mandatoryArgs = [
            'callUUID' => "",
            'rating' => 4.5
        ];
```

or

```
$mandatoryArgs = [
            'callUUID' => $callUUID,
            'rating' => 
        ];
```